internal/controller/storagemgr/storagedriver/storagehandler.go

```diff
@@ -197,6 +245,17 @@ func deviceHandler(writer http.ResponseWriter, request *http.Request) {
 	handler.processAsyncRequest(writer, request)
 }
 
+func GetDeviceHandler(writer http.ResponseWriter, request *http.Request) {
```

Comment:
Do you have a reason to export this function?
Is it possible to merge this function and deviceHandler func into one?

Reply:
Sure, I'll try that.